### DISCUSSION TITLE
[v0.8.0] Event Scheduler: Done Scheduling but Does not execute

### DISCUSSION THREAD

**Discussion by dungdl** [Tracardi/tracardi#815] [Q&A] [answered]
My Tracardi is installed using the Docker Image
I have installed the Event Scheduler plugin and configured the API endpoint, then I have a Node like this:
```{JSON}
{
  "resource": {
    "id": "f121cc4a-c1b5-463d-8f53-ffacf0ef4374",
    "name": "Event scheduler"
  },
  "source": {
    "id": "<hidden>",
    "name": "Data from REST"
  },
  "event_type": "trigger_fb_audience",
  "properties": "{\n    \"id\": \"profile@id\",\n    \"first_seen\": \"profile@aux.ads.first_seen\"\n}",
  "postpone": 10
}
```
The process log show as below:
![Screenshot from 2023-07-19 16-44-48](https://github.com/Tracardi/tracardi/assets/33054551/fd1b7ddc-a7e9-46bb-b536-7af0d5cf156d)
But after 10 seconds it does not send the event as expected.
Besides, according to the doc [Event Scheduler](http://docs.tracardi.com/flow/actions/event_scheduler_action/), the postpone can be like "+10m" but in my version, it cannot be anything else but an Integer.
Did I miss something when configuring the resource?
Thanks for the help!

**Comment by atompie** [comment, marked as answer]
@dungdl Scheduler is part of commercial Tracardi so it will not trigger in open-source version. It is also obsolete. If you use commercial Tracardi use Pause and Resume to schedule events.
